ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v3/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -29,15 +29,13 @@ if (typeof window.FileInputWidget === 'undefined') {
         attachEventHandlers(widget, dragArea, model) {
             super.attachEventHandlers(widget, dragArea, model)
             const customBtn = dragArea?.querySelector(".cmp-adaptiveform-fileinput__widgetlabel")
-            customBtn?.addEventListener("click", (event) => {
-                event.preventDefault();
+            customBtn?.addEventListener("click", () => {
```

Comment:
Can you add test cases for these changes in v3 spec ?

Reply:
https://github.com/adobe/aem-core-forms-components/blob/22d7cc6ae1b48bec540cfbf9de98440fef119a47/ui.tests/test-module/specs/fileinput/fileinputv3.runtime.cy.js#L221

It is already written here.